meta/heart-bits/contao-article-backgrounds-bundle/de.yml

```diff
@@ -0,0 +1,10 @@
+de:
+    title: Artikel-Hintergründe
+    description: >
+        Hintergrundfarben und -bilder für Artikel in Contao.
```

Comment:
Ah, that's why.

@Toflar the spell checker is a little too stupid imho ;). Because of this sentence, which should be correct (no?), you'd have to add `bilder` to the German whitelist. This would also mean if anyone accidentally writes `bilder` instead of `Bilder` in the future, this error will not be catched anymore.

Reply:
I did not invent the spellchecker itself, we're using `aspell` so you may want to add a feature request there. It's still useful to catch most of the issues and to ensure a good quality level. I don't think that writing `Hintergrundfarben und Hintergrundbilder` in that case would be a big problem.

Reply:
Fixed. Thank you both for your input and time!